packages/secure_storage/amplify_secure_storage_test/pubspec.yaml

```diff
@@ -8,8 +8,7 @@ environment:
 
 dependencies:
   test: ^1.22.1
-  amplify_secure_storage_dart:
-    path: ../amplify_secure_storage_dart
+  amplify_secure_storage_dart: any
```

Comment:
do we need an override to get it from local path?

Reply:
Yes. That is handled by `aft link`. Hardcoded paths can cause issues when publishing due to pub's constraint solver. See: https://github.com/aws-amplify/amplify-flutter/pull/3568